draft-dekater-scion-controlplane.md

```diff
@@ -2143,6 +2144,14 @@ To defend against this attack, methods to detect the wormhole attack are needed.
 SCMP External Interface Down ([](#external-interface-down)) and Internal Connectivity Down ([](#internal-connectivity-down)) can potentially be abused by an attacker to to disrupt forwarding of information and/or force the traffic through a different paths. Endpoints should therefore consider them weak hints and apply heuristics to detect fraudulent SCMP messages (e.g. by actively probing whether the affected path is actually down).
 Note that this would be mitigated through authentication of SCMP messages. Authentication is not specified here since it is currently still experimental.
 
+## Attacks on time sources {#time-security}
+
+Care should be taken to maintain coarse time synchronization among Control Service instances and other system components, as discussed in [](#clock-inaccuracy). An adversary that significantly alters the system time of a component can disrupt SCION operations:
+
+- A control service instance: its beaconing process may halt as it cannot verify the validity of received PCBs (see [](#pcb-validity)) or correctly add timestamps to propagated PCBs (see [](#pcb-appending)).
+- An endpoint: the endpoint may fail to verify path segments during path lookup.
+- A router: packets may be dropped if the time offset exceeds the Hop Field expiration time (between 337.5s and a day, see [](#hopfield)).
```

Comment:
The time offset doesn't need to exceed the hopfield's expiration time...

If the clock in the router is N seconds ahead of the control service's clock then the packet at the router will be dropped N seconds before the control service's "intended expiration".